src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTestBase.cs

```diff
@@ -156,14 +156,24 @@ public class NamedCall
                     using var cts = new CancellationTokenSource(8);
                     try {
                         mr.Wait(cts.Token);
-                    } catch (OperationCanceledException) { /* ignore */ }
+                    } catch (Exception ex) {
+                        if (ex is OperationCanceledException)
```

Comment:
Radek's solution here is bit better https://github.com/dotnet/runtime/pull/98508#discussion_r1498998332

Reply:
Removed these changes.